Documentation/operations/upgrade.rst

```diff
@@ -1197,7 +1217,7 @@ Generate the latest ConfigMap
     helm template cilium \
       --namespace=kube-system \
       --set agent.enabled=false \
-      --set config.enabled=true \
+      --set enabled=true \
```

Comment:
this should not be modified since this section referes to upgrade from 1.7 to 1.8, we aren't changing the helm chart of 1.8 branch.

Reply:
fixed